l10n_de_tax_statement/wizard/l10n_de_tax_statement_config_wizard.xml

```diff
@@ -248,7 +248,7 @@
     </record>
 
     <record id="action_account_tax_statement_de_config" model="ir.actions.act_window">
-        <field name="name">Ust-Voranmeldung</field>
+        <field name="name">German Tax Tags</field>
```

Comment:
IMO 'periodic vat returns' would be the better fit here.

Reply:
I think that the "German" string should be specified there because there are some other similar modules that do the same for other localizations, for example the Netherlands Vat Statement module.

Reply:
I am with you on this one. Just not so sure with Tax Tags. Ist this the right expression do you know? The big blue erp 'brother' is using 'periodic vat returns'

Reply:
"Tax Tags" is the Odoo way :smile: 
This is just a configuration menu item, so I don't think 'periodic vat returns' is appropriate. Anyway I don't have any particular problem in changing it. Maybe we can wait and see what other community members think about it.

Reply:
LOL, as I said I not sure about the naming as well. Maybe @tv-openbig has an good idea before merging? ;-)

Reply:
@benbrich can we keep "German Tax Tags" and move forward?

Reply:
Yes for sure.